hw_diag/tasks.py

```diff
@@ -40,64 +64,18 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
     diagnostics['PK'] = public_keys['key']
     diagnostics['AN'] = public_keys['name']
 
-    # Fetch data from miner container.
-    try:
-        diagnostics = fetch_miner_data(diagnostics)
-    except MinerFailedFetchData as e:
-        log.exception(e)
-    except Exception as e:
-        log.exception(e)
-
-    # Get the blockchain height from the Helium API
-    value = None
-    try:
-        value = get_helium_blockchain_height()
-    except KeyError as e:
-        logging.warning(e)
-    except (ConnectTimeout, ReadTimeout):
-        err_str = ("Request to Helium API timed out."
-                   " Will fallback to block height of 1.")
-        logging.exception(err_str)
-    diagnostics['BCH'] = value
-
-    # Check if the miner height
-    # is within 500 blocks and if so say it's synced
-    if diagnostics['BCH'] is None or diagnostics['MH'] is None:
-        diagnostics['MS'] = False
-    elif int(diagnostics['MH']) > (int(diagnostics['BCH']) - 500):
-        diagnostics['MS'] = True
-    else:
-        diagnostics['MS'] = False
-
-    # Calculate a percentage for block sync
-    if diagnostics['BCH'] is None or diagnostics['MH'] is None:
-        diagnostics['BSP'] = None
-    else:
-        diag_mh = int(diagnostics['MH'])
-        diag_bch = int(diagnostics['BCH'])
-        diagnostics['BSP'] = round(diag_mh / diag_bch * 100, 3)
-
     set_diagnostics_bt_lte(diagnostics)
 
-    # Check if the region has been set
-    try:
-        with open("/var/pktfwd/region", 'r') as data:
-            region = data.read()
-            if len(region) > 3:
-                log.info("Frequency: " + str(region))
-                diagnostics['RE'] = str(region).rstrip('\n')
-    except FileNotFoundError:
-        # No region found, put a dummy region in
-        diagnostics['RE'] = "UN123"
-
     # Check the basics if they're fine and set an overall value
     # Basics are: ECC valid, Mac addresses aren't FF, BT Is present,
     # and LoRa hasn't failed
     if (
             diagnostics["ECC"] is True
             and diagnostics["E0"] is not None
             and diagnostics["W0"] is not None
-            and diagnostics["BT"] is True and diagnostics["LOR"] is True
+            and diagnostics["BT"] is True
```

Comment:
Related to: https://github.com/NebraLtd/hm-config/issues/187#issuecomment-1094121136

@shawaj @KevinWassermann94 how do we want to treat PF on the diagnostics page in the immediate term?